composer.json

```diff
@@ -22,7 +22,7 @@
         "zendframework/zend-diactoros": "^1.1",
         "zendframework/zend-expressive-router": "^1.1",
         "zendframework/zend-expressive-template": "^1.0.1",
-        "zendframework/zend-stratigility": ">=1.1.0 < 1.3.0 || >=1.3.1 <2.0.0"
+        "zendframework/zend-stratigility": ">=1.2.0 < 1.3.0 || >=1.3.1 <2.0.0"
```

Comment:
Per a conversation with you in IRC, I'll update this to `^1.3.3`, since we're now using `raiseThrowables()` explicitly within some of the updated code.